java/core/src/test/org/apache/orc/impl/TestSchemaEvolution.java

```diff
@@ -763,8 +763,6 @@ public void testTimestampToDecimal64Evolution() throws Exception {
 
   @Test
   public void testTimestampToStringEvolution() throws Exception {
-    testFilePath = new Path(workDir, "TestSchemaEvolution." +
-                                         testCaseName.getMethodName() + ".orc");
```

Comment:
Is there a reason why you remove this in your new feature PR, @pavibhai ?

Reply:
Oh yeah this is a cleanup, I can raise a separate one for this.

Reply:
Yep. Please make it separately. Then, we can quickly merge it.